configurations/test/env.yml

```diff
@@ -1,6 +1,7 @@
 AUTH0_API_CLIENT: test-auth0-client-id
 AUTH0_API_SECRET: test-auth0-secret
 AUTH0_DOMAIN: test.auth0.com
+DISABLE_AUTH: true
```

Comment:
`DISABLE_AUTH=true` doesn't seem to be needed in this file (it is in the user's own `configurations/default/env.yml` for E2E tests.